### ISSUE TITLE
Hotspot in demo doesnt work on iPhone

### ISSUE THREAD

**Issue by thijsaarts** [google/marzipano#157]
When I take a look at this demo: http://www.marzipano.net/demos/hotspot-styles/index.html on my iPhone, I can't get the hotspot icon that looks like a + to work on the iPhone. It does show the first textballoon the first time I click, but the bigger pop-up doesn't appear. Also not when I click again. 

On desktop the first textballoon appears on hover. On android it appears after the first click. it should show a textballoon with the text "Click to sign up!" the first time you click and a bigger pop-up when you click again showing an image, a text 'Fill the form and we'll open the spam!", an input field, a text 'Just kidding, this doesn't do anything' and a close button. It works fine on Desktop, it works like described here on Android and it doens't show the last pop-up item on iOS. 

Is anyone else familiar with this issue? I can probably edit the code and css to fix it, but I do want to make sure you are aware of this issue or whether me and my co-workers are the only ones experiencing this. 

Thanks

**Comment by vectorjedi** [comment]
Seems like the badge "Click to sign up" is only set to opacity:1 on the hover state of the parent element. That might work on android (since hover is triggered with touchstart, afaik) but doesnt have to on IOS.
Seems like a bug.

**Comment by thijsaarts** [comment]
I've found a solution that is quite simple. In the info.js file you will find several evenlisteners for click events. These will not be recognised on iOS so I added eventlisteners for touchstart events, like so: .addEventListener('touchstart', function() {

I've done this for the icon and the close button. 

I made a lot more changes for my scene, but I will change the original script as well and share it so the demo can be fixed. It's quite easy.

regards,
Thijs
